ext_data/community_models/e_coli_core.json

```diff
@@ -0,0 +1,14786 @@
+{
```

Comment:
is this file needed to be included in the repo? It is huge and readily available from http://bigg.ucsd.edu/static/models/e_coli_core.json

Reply:
similarly for `ext_data/community_models/iIT341.json` it is that large that github cannot show it

Reply:
I removed the 2 `.json` files with a `README` explaining the reasond for this directory

Reply:
We could remove the directory completely if you find it of no use.